docs/modules/technical-guide/pages/user-interface/object-identifiers.adoc

```diff
@@ -0,0 +1,82 @@
+= Object Identifiers
+
+include::common:partial$_header.adoc[]
+
+Various Scout objects like xref:technical-guide:user-interface/widget.adoc[widgets], xref:technical-guide:user-interface/page.adoc[pages] or table columns may have a unique identifier.
+
+- In Scout Classic it is called `classId`.
+- In Scout JS it is called `uuid`.
+
+== Use Cases
+
+The main purpose of such an identifier is to have a stable identifier that is refactoring safe, meaning it won't change when the class or id of the object is renamed.
+This makes it possible to persist references to such Scout objects without having to migrate the persisted data when the objects are renamed.
+
+Another use case of these IDs is UI testing.
+If the url query parameter `inspector` is set to true, these ids are written to the DOM into the attribute `data-uuid`.
+
+NOTE: `inspector` is automatically set to true when the application runs in dev mode (property `scout.devMode` is true) or the url query parameter `debug` is set to true.
+
+Many Scout objects also have an id that is written to the DOM attribute `data-id`.
+This id is not ideal for UI testing because they are eiter dynamic or not unique enough.
+The uuid on the other hand is static and universally unique.
+However, since there may be multiple instances of the same object displayed concurrently,
+the `data-uuid` attribute does not only contain the uuid of the object but the `uuidPath`.
+
+== UUID Path
```

Comment:
Chapter heading needed? If we removed this, the sentence about "multiple instances" would not have to be duplicated.